config/foreman-proxy-content.migrations/190904144224-set_tuning_fact.rb

```diff
@@ -0,0 +1 @@
+scenario[:facts]['tuning'] ||= 'small'
```

Comment:
So we default to small, which is interesting since we'd almost expect that to be the 'defaults' in the installer rather than a profile. It makes me wonder from a naming stand point if we should be calling `small` something like `standard` or `base`.

Reply:
I like standard.

Reply:
The slightly ironic thing is, I would expect default to be 0 tuning, and then some sort of standard tuning profile except that I would then ask as a user why its not the default. So I can live with either name :)

Reply:
Default makes sense as well. Probably slightly more. At least both default and standard are better options than small.